neptune/alpha/types/atoms/datetime.py

```diff
@@ -27,7 +27,7 @@
 class Datetime(Atom):
 
     def __init__(self, value: datetime):
-        self.value = value
+        self.value = value.replace(microsecond=1000*int(value.microsecond/1000))
```

Comment:
We could make this conversion at the very late point in api layer, couldn't we?

Reply:
I think the fact that we use millisecond resolution is part of our domain and should be enforced as fast as possible. It is not responsibility of api layer to make such conversions. If user runs script in offline mode and retrieves values after assignment, should he get different results? Should I put an extra conversion to cover this case ?

Also, it makes less sense IMO in tests. When we send a value to the server and then retrieve this value, we would expect to get the same exact value. By coding conversion here, we can have less tests to be aware of such conversion.